src/test_fixture/event_gen.rs

```diff
@@ -358,13 +370,28 @@ mod tests {
                 1..u32::MAX,
                 1..u32::MAX,
                 1..u32::MAX,
+                1..u32::MAX,
                 report_filter_strategy(),
```

Comment:
Oh, so the idea is to generate a range so that you don't have to swap or clamp?  The other ranges don't allow 0 or `u32::MAX` to be selected; I assume that that is an immaterial difference.